central/enrichment/singleton.go

```diff
@@ -53,6 +57,18 @@ func initialize() {
 	en = New(datastore.Singleton(), ie)
 	cf = fetcher.SingletonManager()
 	manager = newManager(imageintegration.Set(), ne, cf)
+	ctx := sac.WithGlobalAccessScopeChecker(context.Background(), sac.AllowAllAccessScopeChecker())
```

Comment:
should make this a function like before

Reply:
you can just initialize this from within `initializeManager` since it's only used there